job_templates/sag_gnn/app_2/config_fed_client.conf

```diff
@@ -32,7 +32,7 @@
           heartbeat_timeout = 60
 
           # format of the exchange parameters
-          params_exchange_format =  "numpy"
+          params_exchange_format =  "pytorch"
```

Comment:
why change this sag_gnn example?

Reply:
we need to change them back to 2.5.2 , revert https://github.com/NVIDIA/NVFlare/pull/3319